### ISSUE TITLE
Service jobs disappear

### ISSUE THREAD

**Issue by Smeedijzer** [microsoft/Recurring-Integrations-Scheduler#123]
We use RIS as a service on dedicated server, and define jobs when connected to the local service.
The defined jobs seem to be saved correctly, and reconnecting with client shows all jobs.
After a while however we reconnect with the client and all saved jobs seem to have disappeared 

Side question: We have a schedule.xml saved separately, can this be used to restore the job settings?

**To Reproduce**
Not sure how to reproduce, might be related to server reboot triggered by Windows update

**Expected behavior**
Service's saved jobs remain until manually deleted

**Recurring Integration Scheduler version (please complete the following information):**
 - Version 3.2.2.0

**Dynamics 365 version (please complete the following information):**
 - Application version: D365 Fin 10.0.13
 - Platform version: PU37

**Comment by drtolive** [comment]
I am experiencing the same issue after a server restart.  Has anyone else seen this?

**Comment by VadimGoncharenko** [comment]
I do, quite often

**Comment by drtolive** [comment]
> I do, quite often

Have you found a fix or workaround besides just re-adding the jobs?

**Comment by TomekMelissa** [comment]
schedule.xml file is the input file with jobs definition for the RIS service. It needs to be stored in the same folder as RIS service executable and accessible for the service user. The whole RIS app, apart from the standalone scheduler feature, is merely a UI for configuring jobs and saving them as schedule.xml file.
It is possible to add or delete job(s) to and from running RIS service. If you do this through RIS app it will try to save the current state of jobs to schedule.xml file every time you exit the app in a standard way. If you will kill the app (not service) and schedule.xml is not saved then it is possible that the current (not persisted) list of jobs in running service is different from the list of jobs in schedule.xml file. After you restart the service it will read schedule.xml file for a list of jobs.

**Comment by drtolive** [comment]
Thank you so much for the update.  I did verify that the schedule.xml was in the working folder and when prompted to save I saved to that file after adding jobs via the RIS app.    I restarted today and the jobs remained.  I believe it was as you indicated that I may have added the job from the app and when saving the schedule I may have saved it to the wrong directory.

Thank you again for responding so quickly.  I am so pleased to have found the application and it is working very well.
